content/solutions/index.html.haml

```diff
@@ -1,11 +1,37 @@
 ---
 layout: simplepage
 title: "Jenkins Use-cases"
+noanchors: true
 ---
 
-%ul
-  - site.solutions.keys.sort.each do |key|
+:css
+  .card h3 {
+    font-size: 1.3rem;
+    min-height: 3rem;
+  }
+  .card img {
+    height:96px;
```

Comment:
Would be nice to have spacing consistent for css normally one space after the :. Harder for IDE and linter because it’s embedded in Haml I assume